src/diff/line.js

```diff
@@ -3,6 +3,11 @@ import {generateOptions} from '../util/params';
 
 export const lineDiff = new Diff();
 lineDiff.tokenize = function(value) {
+  if(this.options.stripTrailingCr) {
+    // remove all CR (\r) characters from the input string
+    value = value.replace(/\r+\n/g, '\n');
```

Comment:
The comment, code, and analogy to GNU `diff`'s `--strip-trailing-cr` option all disagree about the behaviour here:

* `--strip-trailing-cr` only removes at most ONE `\r` before an `\n`
* the code here removes a sequence of any number of consecutive `\r` characters before a `\n`
* the comment implies that all `\r` characters will be removed, even if they don't appear before a `\n`

We should tweak the behaviour to be consistent with the GNU diff option whose name we're copying, and we should make the comment accurately reflect the behaviour.

Reply:
That's a good point! Have an update!